Sources/App/configure.swift

```diff
@@ -2,24 +2,27 @@ import Vapor
 import Stevenson
 
 private let jiraProjects = [
-    "NRX"  : 16911, // Enrolment and Integrity
-    "CE"   : 16937, // Consultation Experience
-    "AV"   : 16942, // Core Experience / Avalon
-    "CW"   : 16832, // Consumer Web
-    "CNSMR": 16968, // Consumer Apps (Native/Core)
-    "MON"  : 10103, // HealthCheck
-    "GW"   : 16949, // Triage UI
-    "PRSCR": 16840, // Prescriptions
-    "SDK"  : 16975, // SDK
-    "APPTS": 16875, // Booking/Appointments
-    "TEL"  : 16857, // Telus
-    "PRO"  : 16980, // Professional Services
-    "PAR"  : 17098, // Partnerships
-    "MN"   : 17031, // Monitor
-    "TES"  : 17074, // Tests & Kits
-    "WH"   : 17112, // Women's Health
-    "IDM"  : 16903, // Identity Platform / Identity Management
-    "CM"   : 16920, // Condition Management
+    "APPTS" : 16875, // Booking/Appointments
+    "AV"    : 16942, // Core Experience / Avalon
+    "CE"    : 16937, // Consultation Experience
+    // TODO: [CNSMR-2402] Re-enable CM board once they've migrated all their tickets from private instance to main one
+    // "CM"    : 16920, // Condition Management
+    "CNSMR" : 16968, // Consumer Apps (Native/Core)
+    "COREUS": 17127, // Babylon US Core Product
+    "CW"    : 16832, // Consumer Web
+    "GW"    : 16949, // Triage UI
+    // TODO: [CNSMR-2402] Re-enable IDM and MN once those boards have migrated to use the standard setup for their JIRA fields
+    // "IDM"   : 16903, // Identity Platform / Identity Management
+    // "MN"    : 17031, // Monitor –
+    "MON"   : 10103, // HealthCheck
+    "NRX"   : 16911, // Enrolment and Integrity
+    "PAR"   : 17098, // Partnerships
+    "PRO"   : 16980, // Professional Services
+    "PRSCR" : 16840, // Prescriptions
+    "SDK"   : 16975, // SDK
+    "TEL"   : 16857, // Telus
+    "TES"   : 17074, // Tests & Kits
```

Comment:
They are now “Test Kits” :)

Reply:
Ha! I got it from the official name of the JIRA board itself (see key `name` in https://babylonpartners.atlassian.net/rest/api/3/project/TES ) I guess they should update it there too then 😉

Reply:
Yeah it's just been changed recently, so not updated everywhere.